src/deck.py

```diff
@@ -1,7 +1,20 @@
 from card import Card
-
+import random
+import uuid
 
 class Deck:
+    suits = ['Hearts', 'Diamonds', 'Clubs', 'Spades']
```

Comment:
This will cause problems for the UI since the expectation is that that the suit is a single letter.